french/manger/trou/trou.md

```diff
@@ -2,4 +2,7 @@ Après être passé par le trou vous êtes tout gluant...
 Un beau soleil brille et vous sêche, 
 mais le chamallow séché caramélise et vous rend de plus en plus gêné dans vos mouvements.
 
-
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.